test/aws/centos_repos.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Note: OpenShift repositories under CentOS may be shipped through the
+# "centos-release-openshift-origin" package which configures the repository.
+# This task matches the file names provided by the package so that they are
+# not installed twice in different files and remains idempotent.
+- name: Ensure libselinux-python is installed
```

Comment:
This has nothing to do with repos.  If it is still needed (I don't think so), it should be moved to openshift_node.

Reply:
Will do. In my local testing it works fine without it.